CorsixTH/Src/th_map.cpp

```diff
@@ -619,8 +619,16 @@ void THMap::setParcelOwner(int iParcelId, int iOwner)
                     }
                 }
             }
-            _checkDividers(pNode, pOriginalNode, iX, 1, 2, iParcelId);
-            _checkDividers(pNode, pOriginalNode, iY, 128, 1, iParcelId);
+            bool isSplitTile = _checkDividers(pNode, pOriginalNode, iX, 1, 2, iParcelId);
```

Comment:
Any particular reason you don't fold the _check..(..) thingie directly in the 'if' ? the boolean is used one time, and "bool isSplitTile = " is longer than "if (".